README.md

```diff
@@ -474,26 +477,27 @@ based on evidence that it will make money in the future.
     company.
 - 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company is the
   present value investors believe the company has.
-  If the company is doing well, growing revenue or has indications of future revenue (like
-  growing users or traction in a promising market), the company’s valuation will usually be
-  on the rise. That is, the **price** for an investor to buy one share of the company would be
-  increasing.
+  If the company is doing well, bringing in revenue or establishing indications of future
+  revenue (like a growing number of users or traction in a promising market), the company’s
+  valuation will usually be on the rise.
+  With that rising valuation, the **price** for an investor to buy one share of the company
+  increases.
 - ❗️ Of course, things do not always go well, and the valuation of a company does not always
   go up. It can happen that a company fails entirely and everyone’s shares become worthless,
-  or that the valuation is below expectations and [certain kinds of shares](#classes-of-stock)
+  or that the valuation is lower than expected and [certain kinds of shares](#classes-of-stock)
   become worthless while other kinds have some value.
   When investors and leadership in a company expect the company to do better than it
   actually does, it can have a lot of disappointing consequences for shareholders.
 
-As a rough illustration of how ownership of a venture-backed company evolves as funding is
-raised, a couple visualizations of shares are below.
-One is a completely imaginary but well-performing startup, and the other is loosely based
-on
+The visualizations below are rough illustrations of how ownership of a venture-backed
+company evolves as funding is raised.
+One figure imagines changes to ownership in a well-performing startup, and the other is
```

Comment:
figure -> scenario

(they are not separate figures)

Reply:
Fixed.